misc/assets/inforequests.svg

```diff
@@ -0,0 +1,248 @@
+    <marker id="relation-optional" orient="auto-start-reverse" markerWidth="6" markerHeight="6" refX="6" refY="3">
+      <circle cx="3" cy="3" r="2.5" fill="white" stroke="black" stroke-width="1" />
+    </marker>
+    <marker id="link_attribute" orient="-90" markerWidth="10" markerHeight="6" refX="5" refY="0">
+      <path d="M 1 0.5 A 1.5 2, 0, 0 0, 8.5 0.5" fill="white" stroke="black" />
+    </marker>
+    <marker id="dependency" markerWidth="6" markerHeight="6" refX="3" refY="0">
+      <polygon points="0,6 6,6 3,0" />
+    </marker>
+    <g id="model">
+      <rect class="box" x="-100" y="-20" width="200" height="40" fill="none" stroke="black" stroke-width="1pt" />
+    </g>
+    <g id="model-narrow">
+      <rect class="box" x="-50" y="-20" width="100" height="40" fill="none" stroke="black" stroke-width="1pt" />
+    </g>
+  </defs>
+
+  <g transform="translate(100 100)">
+    <use href="#model-narrow" class="external"/>
+    <text class="caption">User</text>
+    <path class="relation from-one to-many" d="M 0,20 0,80" />
+    <path class="relation from-one to-many" d="M 50,0 100,0" />
+  </g>
+  <g transform="translate(100 200)">
+    <use href="#model-narrow" />
+    <text class="caption">Inforequest</text>
+    <path class="relation from-optional to-many" d="M 0,20 0,180" />
+    <path class="relation from-one to-many" d="M 20,-20 20,-40 450,-40 450,-80" />
+    <path class="relation from-one to-many" d="M 50,0 450,0" />
+    <text class="note" x="430" y="0">1+</text>
+  </g>
+  <g transform="translate(150 400)">
+    <use href="#model" class="external" />
+    <text class="caption">E-mail Message</text>
+    <path class="relation from-optional to-optional" d="M 100,0 400,0" />
+  </g>
+  <g transform="translate(300 100)">
+    <use href="#model" />
+    <text class="caption">Inforequest Draft</text>
+    <path class="relation from-many to-optional" d="M 0,-20 0,-40 500,-40 500,-20" />
+  </g>
+  <g transform="translate(300 300)">
+    <use href="#model" />
+    <text class="caption">Inforequest E-mail</text>
+    <path class="relation from-one to-link_attribute" d="M -100,0 -200,0" />
+  </g>
+  <g transform="translate(600 100)">
+    <use href="#model" />
+    <text class="caption">Action Draft</text>
+    <path class="relation from-many to-optional" d="M 0,20 0,80" />
+    <path class="relation from-many to-many" d="M 100,0 150,0" />
+  </g>
+  <g transform="translate(600 200)">
+    <use href="#model-narrow" />
+    <text class="caption">Branch</text>
+    <path class="relation from-many to-one" d="M 25,-20 25,-60 175,-60 175,-80" />
+    <path class="relation from-many to-one" d="M 50,0 100,0" />
+    <path class="relation from-one to-many" d="M -25,20 -25,180" />
+    <text class="note" x="-10" y="180">1+</text>
+    <text class="note left-rotation" x="-135" y="-30">Contained</text>
+    <path class="relation from-many to-optional" d="M 25,20 25,180" />
+    <text class="note right-rotation" x="70" y="-30">Advanced by</text>
```

Comment:
Pri `note left-rotation` a `note right-rotation` by bolo dobre definovat v CSS zarovnanie textu. Nech `x` nemusis rucne menit, ked zmenis dlzku textu. Ale nech ako `x` mozes uviest jeden fixny bod na zaciatku/konci ciary a text sa zarovna sam.

In general, vsetky cisla co sa daju by mali byt implicitne zjavne, nie explicitne vyratane na papieri a potom prepisane (resp. natipovane). Aby sa to potom lahsie udrziavalo. 

Edit: Btw, iba jedna z "Contained" a "Advanced by" je definovana v tomto modeli. Druha je definovana v druhom modeli. Ked ciaru presunies do spravneho modelu, tak v oboch pripadoch bude text zacinat blizko boxu, cize v oboch pripadoch by `x` malo byt nizke.

Reply:
:ok: